backend/src/govsg/models/campaign-govsg-template.ts

```diff
@@ -13,7 +13,11 @@ import { GovsgTemplate } from './govsg-template'
 //   but we're creating a pivot table to not have to modify campaign table,
 //   which, in tandem with the old set up of templates for other channels,
 //   will greatly confuse readers
-@Table({ tableName: 'campaign_govsg_template', underscored: true })
+@Table({
+  tableName: 'campaign_govsg_template',
+  underscored: true,
+  timestamps: false,
```

Comment:
why don't we want these columns

Reply:
This table is just a pivot table, not that impt for these timestamps me thinks